client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,12 +54,39 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
-				<FormButton
-					onClick={ ( event ) => {
-						gaRecordEvent( 'Me', 'Clicked On 2fa Get Started Button' );
-						this.props.onSuccess( event );
-					} }
-				>
+				<div>
+					<FormLabel className="security-2fa-initial-setup__label">
+						<FormRadio
+							name="auth_method"
+							value="app-based"
+							defaultChecked={ isAppSelected }
+							onChange={ this.setAuth.bind( this ) }
+						/>
+						<Gridicon icon="phone" />
+						<span>Set up using an app</span>
+						<p>
+							Use an application on your phone to get two-factor authentication codes when you log
+							in.
+						</p>
+					</FormLabel>
+
+					<FormLabel className="security-2fa-initial-setup__label">
+						<FormRadio
+							name="auth_method"
+							value="sms-settings"
+							defaultChecked={ ! isAppSelected }
+							onChange={ this.setAuth.bind( this ) }
+						/>
+						<Gridicon icon="chat" />
+						<span>Set up using SMS</span>
+						<p>
+							WordPress.com will send you a SMS with a two-factor authentication codes when you log
+							in.
+						</p>
+					</FormLabel>
+				</div>
+
+				<FormButton onClick={ this.handleClick.bind( this ) }>
```

Comment:
Instead of binding to `this`, we can just declare `handleClick` as an arrow function above. That will automatically bind it to `this`.